YAFCmodel/Data/DataUtils.cs

```diff
@@ -158,7 +158,7 @@ public int Compare(T x, T y)
         
         public static Solver CreateSolver(string name)
         {
-            var solver = Solver.CreateSolver(name, "GLOP_LINEAR_PROGRAMMING");
+            var solver = Solver.CreateSolver("GLOP_LINEAR_PROGRAMMING");
```

Comment:
It seems the two-argument constructor to `Solver.CreateSolver` got removed at some point, based on the examples it looks like there is only one argument to the constructor now: the target algorithm.